staff/sergio-luz/pre-course/cv/index.html

```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Jaime Gonzalez: curriculum vitae</title>
+</head>
+<body>
+    <header>
+        <h1>Jaime Gonzalez / Web Developer</h1>
+        <address>Gran via de les Corts Catalanes 587</address>
+        <address>08054, Barcelona, España</address>
+        <a href="mailto:JaimeGonzalez@gmail.com">JaimeGonzalez@gmail.com</a>
+    </header>
+    <hr>
+
+    <section>
+        <h2>Professional Experience</h2>
```

Comment:
Maybe you could separate the sections (professional experience, studies, etc) into different sections within a main section